istio.io/configmap-www-golang.yaml

```diff
@@ -18,6 +18,8 @@ data:
     <html><head>
       <meta name="go-import" content="istio.io/istio git https://github.com/istio/istio">
       <meta name="go-source" content="istio.io/istio     https://github.com/istio/istio https://github.com/istio/istio/tree/master{/dir} https://github.com/istio/istio/blob/master{/dir}/{file}#L{line}">
+      <meta name="go-import" content="istio.io/istio/vendor git https://github.com/istio-releases/vendor">
```

Comment:
prep for https://github.com/istio/istio/pull/3348

Reply:
Would we switch to `github.com/istio/vendor` at some point once commited vendor dependency scheme is vetted?

Reply:
yes I think we could move    istio-releases/vendor -> istio/vendor-istio and istio-releases/fortio-vendor -> istio/vendor-fortio

Reply:
fixed in #53